Jenkinsfile

```
@@ -30,6 +30,34 @@ boolean doc_only_change() {
     return rc == 1
 }
 
+boolean release_candidate() {
+    if (!sh(label: "Determine if building (a PR of) an RC",
+          script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
+                  "grep -i '[0-9]rc[0-9]' TAG",
+          returnStatus: true)) {
+        return true
+    }
+    return false
+}
+
+def scons_faults_enabled() {
```

```suggestion
def scons_faults_args() {
```

Seems like a more accurate name.